docs/core_docs/package.json

```diff
@@ -7,7 +7,7 @@
     "start": "yarn build:typedoc && rimraf ./docs/api && NODE_OPTIONS=--max-old-space-size=7168 docusaurus start",
     "build": "yarn clean && yarn build:typedoc && rimraf ./build && NODE_OPTIONS=--max-old-space-size=7168 DOCUSAURUS_SSR_CONCURRENCY=4 docusaurus build",
     "build:vercel": "yarn clean && yarn build:typedoc:vercel && rimraf ./build && NODE_OPTIONS=--max-old-space-size=7168 DOCUSAURUS_SSR_CONCURRENCY=4 docusaurus build",
-    "build:typedoc": "yarn run turbo:command build --filter=api_refs",
+    "build:typedoc": "yarn workspace api_refs build",
```

Comment:
Not with `turbo`?

Reply:
correct, this was the root of our problems with building on vercel, but when I fixed that I only removed it from the `build:typedoc:vercel` script.

Reply:
Ah ok